packages/react/src/context/slash-id-context.tsx

```diff
@@ -105,15 +105,23 @@ SlashIDContext.displayName = "SlashIDContext";
 
 const STORAGE_TOKEN_KEY = "@slashid/USER_TOKEN";
 
-const createStorage = (storageType: StorageOption) => {
+const createStorage = (storageType: StorageOption, options: { anonymousUsersEnabled: boolean }) => {
```

Comment:
IMO this must be handled one level above, as the SlashID context provider is the owner of this business logic.